apriltag_ros/src/continuous_detector.cpp

```diff
@@ -127,10 +127,11 @@ bool ContinuousDetector::isTagInFOV(const tf::Transform& tag_pose) const
       << " has target at pixel coordinate x: " << image_pt.x << ", y: " << image_pt.y);
 
   // check if the image is within the target fov
+  const auto cam_resolution = camera_model_.reducedResolution();
```

Comment:
Does it make sense to do this call again here, or should I make `cam_resolution_` a private member and set it in the image callback, then just read that value here? The image callback always runs before this method so that should work fine. Thoughts?

Reply:
Yes, that makes sense.

Just don't assume the camera model doesn't change, so update the camera model every time you get a CameraInfo message, which I believe it's what we already do. So basically, right after updating the camera model, you can also cache the reduced resolution.

In the camera model object they already cache the rectified ROI, but not the reduced resolution, likely because it's relatively trivial, just a few math operations. So I don't have a strong opinion, and it'd be fine if you don't create any attribute.

I'd rename the variable to `camera_resolution_`, no need to use a shorter name IMHO:
```suggestion
  const auto camera_resolution = camera_model_.reducedResolution();
```

Or use `camera_resolution_` for an attribute.

Reply:
Ok makes sense. I'll just rename then to avoid making more changed

Reply:
changed.